docs/content/guides/rows/row-sorting.md

```diff
@@ -83,93 +145,502 @@ import 'handsontable/dist/handsontable.full.min.css';
-        ['Mitsubishi', 'iMiEV', '', 'Japan', '★★'],
-        ['Ford', 'Focus EV', '', 'USA', '★★'],
-        ['Mitsubishi', 'iMiEV Sport', '', 'Japan', '★★'],
-        ['Tesla', 'Roadster', 'DarkStar', 'USA', '★★★★★'],
-        ['Volkswagen', 'e-Golf', '', 'Germany', '★★'],
-        ['Volkswagen', 'E-Up!', '', 'Germany', '★★'],
-        ['Ford', 'C-Max Energi', '', 'USA', '★'],
-        ['BYD', 'Denza', '', 'China', '★★★'],
-        ['BYD', 'e5', '', 'China', '★★★'],
-        ['BYD', 'e6', '', 'China', '★★★★']
+      {
+        brand: 'Tagcat',
+        model: 'Classic Vest',
+        price: 327,
+        sellDate: '11/10/2020',
+        sellTime: '01-2331',
+        inStock: true,
+      },
+      {
+        brand: 'Zoomzone',
+        model: 'Cycling Cap',
+        price: 717,
+        sellDate: '03/05/2020',
+        sellTime: '88-2768',
+        inStock: false,
+      },
+      {
+        brand: 'Meeveo',
+        model: 'Full-Finger Gloves',
+        price: 314,
+        sellDate: '27/03/2020',
+        sellTime: '51-6775',
+        inStock: true,
+      },
+      {
+        brand: 'Buzzdog',
+        model: 'Mountain Frame',
+        price: 268,
+        sellDate: '29/08/2020',
+        sellTime: '44-4028',
+        inStock: false,
+      },
+      {
+        brand: 'Katz',
+        model: 'Half-Finger Gloves',
+        price: 204,
+        sellDate: '02/10/2020',
+        sellTime: '08-2758',
+        inStock: true,
+      },
+    ]}
+      columns={[
+        {
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          type: 'text',
+          data: 'model',
+        },
+        {
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US'
+          },
+        },
+        {
+          type: 'date',
+          data: 'sellDate',
+        },
+        {
+          type: 'text',
+          data: 'sellTime',
+        },
+        {
+          type: 'checkbox',
+          data: 'inStock',
+        },
       ]}
-      colHeaders={['Brand', 'Model', 'Code name', 'Country of origin', 'Rank']}
-      columnSorting={true}
+      colHeaders={[
+        'Brand',
+        'Model',
+        'Price',
+        'Date',
+        'Time',
+        'In stock',
+      ]}
+      columnSorting={{
+        // at initialization, sort rows by column 1, in descending order
+        initialConfig: {
+          column: 1,
+          sortOrder: 'desc',
+        },
+      }}
+      rowHeaders={true}
       height="auto"
+      width="auto"
       licenseKey="non-commercial-and-evaluation"
     />
   );
 };
 
 /* start:skip-in-preview */
-ReactDOM.render(<ExampleComponent />, document.getElementById('example1'));
+ReactDOM.render(<YourHandsontableComponent />, document.getElementById('example1'));
 /* end:skip-in-preview */
 ```
 :::
+
 :::
 
+Only the table view is sorted. Your source data remains in the original order.
```

Comment:
What's "table view"? I think we're introducing a new nomenclature here for a thing that already has the name "visual data" or similar. To be determined.

Reply:
Done: [b545d9e](https://github.com/handsontable/handsontable/pull/10183/commits/b545d9e5f9253748351c0e786b8d3d45d4af535d)